.github/workflows/main.yml

```diff
@@ -13,9 +13,9 @@ jobs:
     with:
       linux_5_9_enabled: false
       linux_5_10_enabled: false
-      linux_6_0_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
-      linux_nightly_6_0_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
-      linux_nightly_main_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable"
+      linux_6_0_arguments_override: "--explicit-target-dependency-import-check error -Xswiftc -require-explicit-sendable -Xswiftc -enable-upcoming-feature -Xswiftc MemberImportVisibility"
```

Comment:
Any reason we can't just add this to our default settings in the manifest? https://github.com/grpc/grpc-swift/blob/e390c85ad880004513221555884ae77b08714f1e/Package.swift#L48